src/native/libs/System.Security.Cryptography.Native/opensslshim.c

```diff
@@ -116,8 +116,21 @@ static void OpenLibraryOnce(void)
         DlOpen(MAKELIB("10"));
     }
 
-    // FreeBSD uses a different suffix numbering convention.
-    // Current supported FreeBSD releases should use the order .11 -> .111
+#ifdef __FreeBSD__
```

Comment:
I thought that ".11" came from a Linux, but that seems to be ".10"; Fedora was re-aligned with libcrypto.so.1.1 for OpenSSL 1.1.1 releases.  So it's fine that "11" and "111" are out of the list for Linuxes.

However, I am mildly concerned that there's a libcrypto.so.10 in the FreeBSD versioning flow that will have higher priority over the .12/.30/.11/.111 here (since .10 is the last thing checked before this `#ifdef`.  If there's no practical concern there, no worries; but if it is a practical concern then these version numbers should probably be interspersed with their Linux soname bretheren

Reply:
Thanks for pointing this out! 
I looked into this edge case and I was unable to find a base version of FreeBSD where this might occur.  The in-support versions of FreeBSD use either .12 (from ports) or .30 (from base) for OpenSSL3. 

AFAIK for this to happen it would be for a consumer that is using a historic version of FreeBSD or out of date ports tree.